scene/resources/physics_material.cpp

```diff
@@ -44,29 +47,112 @@ void PhysicsMaterial::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("is_absorbent"), &PhysicsMaterial::is_absorbent);
 
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "preset", PROPERTY_HINT_ENUM, "Generic,Brick,Concrete,Ceramic,Gravel,Carpet,Glass,Plaster,Wood,Metal,Rock,Custom"), "set_preset", "get_preset");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "friction", PROPERTY_HINT_RANGE, "0,1,0.01,or_greater"), "set_friction", "get_friction");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "rough"), "set_rough", "is_rough");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "bounce", PROPERTY_HINT_RANGE, "0,1,0.01,or_greater"), "set_bounce", "get_bounce");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "absorbent"), "set_absorbent", "is_absorbent");
+
+	BIND_ENUM_CONSTANT(PRESET_GENERIC);
+	BIND_ENUM_CONSTANT(PRESET_BRICK);
+	BIND_ENUM_CONSTANT(PRESET_CONCRETE);
+	BIND_ENUM_CONSTANT(PRESET_CERAMIC);
+	BIND_ENUM_CONSTANT(PRESET_GRAVEL);
+	BIND_ENUM_CONSTANT(PRESET_CARPET);
+	BIND_ENUM_CONSTANT(PRESET_GLASS);
+	BIND_ENUM_CONSTANT(PRESET_PLASTER);
+	BIND_ENUM_CONSTANT(PRESET_WOOD);
+	BIND_ENUM_CONSTANT(PRESET_METAL);
+	BIND_ENUM_CONSTANT(PRESET_ROCK);
+	BIND_ENUM_CONSTANT(PRESET_CUSTOM);
+}
+
+void PhysicsMaterial::set_preset(Preset p_preset) {
+	if (preset == p_preset && p_preset != PRESET_CUSTOM) {
+		return;
+	}
+	preset = static_cast<Preset>(p_preset);
+	switch (p_preset) {
+		case PRESET_GENERIC:
+			friction = 0.8;
+			bounce = 0.8;
+			break;
+		case PRESET_BRICK:
+			friction = 0.6;
+			bounce = 0.05;
+			break;
+		case PRESET_CONCRETE:
+			friction = 0.65;
+			bounce = 0.1;
+			break;
+		case PRESET_CERAMIC:
+			friction = 0.15;
+			bounce = 0.7;
+			break;
+		case PRESET_GRAVEL:
+			friction = 0.7;
+			bounce = 0.15;
+			break;
+		case PRESET_CARPET:
+			friction = 0.8;
+			bounce = 0.05;
+			break;
+		case PRESET_GLASS:
+			friction = 0.2;
+			bounce = 0.6;
+			break;
+		case PRESET_PLASTER:
+			friction = 0.55;
+			bounce = 0.25;
+			break;
+		case PRESET_WOOD:
+			friction = 0.45;
+			bounce = 0.15;
+			break;
+		case PRESET_METAL:
+			friction = 0.6;
+			bounce = 0.1;
+			break;
+		case PRESET_ROCK:
+			friction = 0.7;
+			bounce = 0.2;
+			break;
+		case PRESET_CUSTOM:
+			break;
+		default:
+			set_preset(PRESET_CUSTOM);
+			break;
+	}
+	emit_changed();
 }
 
 void PhysicsMaterial::set_friction(real_t p_val) {
 	friction = p_val;
+	preset = static_cast<Preset>(PRESET_CUSTOM);
 	emit_changed();
 }
 
 void PhysicsMaterial::set_rough(bool p_val) {
 	rough = p_val;
+	preset = static_cast<Preset>(PRESET_CUSTOM);
 	emit_changed();
 }
 
 void PhysicsMaterial::set_bounce(real_t p_val) {
 	bounce = p_val;
+	preset = static_cast<Preset>(PRESET_CUSTOM);
 	emit_changed();
 }
 
 void PhysicsMaterial::set_absorbent(bool p_val) {
 	absorbent = p_val;
+	preset = static_cast<Preset>(PRESET_CUSTOM);
 	emit_changed();
 }
+
+PhysicsMaterial::PhysicsMaterial() {
+	preset = PRESET_GENERIC;
+	friction = 0.8;
+	bounce = 0.8;
+}
```

Comment:
Can't change the default friction and bounce here, that is a major compatibility breakage, it should remain 1.0 and 0.0 respectively